src/main/java/org/study/configuration/JDBCConnection.java

```diff
@@ -0,0 +1,44 @@
+package org.study.configuration;
+
+import org.apache.commons.dbcp2.BasicDataSource;
+import org.apache.log4j.Logger;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.TimeZone;
+
+public class JDBCConnection {
+
+    private static final Logger LOG = Logger.getLogger(JDBCConnection.class);
+
+    private final static String URL = "jdbc:mysql://localhost:3306/cinemaproject?verifyServerCertificate=false" +
+            "&useSSL=false&requireSSL=false&useLegacyDatetimeCode=false&amp&serverTimezone="
```

Comment:
are you sure you need all this properties? Propose to add encoding properties

Reply:
Will be fix in next MR